web/web-test-runner.config.mjs

```diff
@@ -40,21 +42,31 @@ export default /** @type {import("@web/test-runner").TestRunnerConfig} */ ({
       'process.env.API_PORT': '3000',
     }),
   ],
-  /** Compile JS for older browsers. Requires @web/dev-server-esbuild plugin */
-  // esbuildTarget: 'auto',
 
   /** Amount of browsers to run concurrently */
-  // concurrentBrowsers: 2,
+  concurrentBrowsers: 2,
 
   /** Amount of test files per browser to test concurrently */
-  // concurrency: 1,
+  concurrency: 10,
 
   /** Browsers to run tests on */
-  // browsers: [
-  //   playwrightLauncher({ product: 'chromium' }),
-  //   playwrightLauncher({ product: 'firefox' }),
-  //   playwrightLauncher({ product: 'webkit' }),
-  // ],
+  browsers: [
```

Comment:
Why not include safari with `webkit`?

Reply:
Oh, thought I replied here. I can't test with webkit locally (missing dependencies to allow me to do so), and it was hitting issues. I'll add it in a follow-up.